pages/CreatingAnExploration.html

```diff
@@ -44,10 +46,21 @@
       this should be the language that the exploration is written in, not the
       language that the exploration is trying to teach.)
     </li>
+    <li>
+      <strong>Name of first Card.</strong> You might want to give your very first 
+      card a name, this makes it possible by filling in the name here.
+    </li>
+    <li>
+      <strong>Tags.</strong> This helps your learner discover your exploration when
+      searching. You might like to give those names that are closely related to 
+      your exploration name or purpose.
+    </li>
   </ul>
 
   <p>
-    Once you've done that, click the green button in the bottom right.
+    Once you've done that, there are other options, those you shall use as you go
+    on creating your exploration. Once you are through, scroll to the buttom of 
```

Comment:
through --> done

You don't need to "scroll to the bottom of the page". Just say: "Once you are done, click the green 'Save Draft' button in the navbar to save your changes."